HWLesson6_8/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css">
+    <title>E-Store</title>
+</head>
+<body>
+    <!-- //navbar -->
+    <div class="topnav" id="myTopnav">
+        <a href="#home" class="active">Home</a>
+        <div class="dropdown">
+            <button class="dropbtn">Products
+            <i class="fa fa-caret-down"></i>
+            </button>
+            <div class="dropdown-content">
+                <a href="images/bosch.jpg">Bosch</a>
+                <a href="images/rowenta.jpg">Rowenta</a>
+                <a href="images/dyson.jpg">Dyson</a>
+                <a href="images/aeg.jpg">AEG</a>
+            </div>
+        </div>
+        
+        <a href="#contact">Contact</a>
+        <a href="#about">About</a>
+    </div>
+
+    <!-- Live Chat button -->
+    <button class="open-button" onclick="openForm()">Live Chat</button>
+
+    <div class="chat-popup" id="myForm">
+        <form action="/action-page.php" class="form-container">
+            <h2>Support</h2>
+
+            <label for="msg"><b>Message</b></label>
+            <textarea placeholder="Type something.." name="msg" required></textarea>
+
+            <button type="submit" class="btn">Send</button>
+            <button type="button" class="btn cancel" onclick="closeForm()">Close</button>
+        </form>
+    </div>
+
+    <script>
+        function openForm() {
+            document.getElementById("myForm").style.display = "block";
+        }
+
+        function closeForm() {
+            document.getElementById("myForm").style.display = "none";
+        }
+    </script>
+
+    <!-- //products -->
+    <div class="wrapper">
+        <div class="nestedCleaner">

```

Comment:
You can generate a function which generate this template using JS or string literals, and display the items using for loops.